website/docs/docs/collaborate/dbt-explorer-faqs.md

```diff
@@ -0,0 +1,176 @@
+---
+title: "dbt Explorer FAQs"
+sidebar_label: "dbt Explorer FAQs"
+description: "Learn more with the FAQs about dbt Explorer, how it works, how to interact with it, and more."
+pagination_next: null
+---
+
+[dbt Explorer](/docs/collaborate/explore-projects) is dbt Cloud’s new knowledge base and lineage visualization experience. It offers an interactive and high-level view of your company’s entire data estate, where you can do dive deeps for the context you need to understand and improve lineage so your teams can trust the data they’re using to make decisions.
```

Comment:
Should we exclude the word "new" so that we don't have to update this later?
```suggestion
[dbt Explorer](/docs/collaborate/explore-projects) is dbt Cloud’s new knowledge base and lineage visualization experience. It offers an interactive and high-level view of your company’s entire data estate, where you can do deep dives for the context you need to understand and improve lineage so your teams can trust the data they’re using to make decisions.
```

Reply:
i was considering that too. however, i think this'll get updated a few more times before GA release so i'm ok with it there ... for now